CHANGELOG.md

```diff
@@ -54,6 +54,7 @@
 - Fix and improve flaky Instrumentation tests.
 - Improve rendering of Instrumentations.
 - Remove unnecessary borehole property `date` from csv import as well as in the database.
+- Fix scrolling behaviour for _lithological_descriptions_ and _facies_descriptions_.
```

Comment:
REQ: Dieser Text muss in die _Unreleased_-Version verschoben werden.

Reply:
Oh hab nicht gecheckt dass da zwei Titel sind !